toucan_connectors/mysql/mysql_connector.py

```diff
@@ -151,16 +155,12 @@ def _sanitize_ssl_params(self) -> dict[str, Any]:
         params = {}
         if self.ssl_mode in (SSLMode.VERIFY_CA, SSLMode.VERIFY_IDENTITY):
             for ssl_opt in ('ssl_ca', 'ssl_key', 'ssl_cert'):
-                assert (
-                    opt := getattr(self, ssl_opt)
-                ) is not None, (
-                    f'{ssl_opt} must be specified if ssl_mode is VERIFY_CA or VERIFY_IDENTITY'
-                )
                 try:
-                    params[ssl_opt] = sanitize_spaces_pem(opt.get_secret_value())
-
+                    if (opt := getattr(self, ssl_opt)) is not None:
+                        params[ssl_opt] = sanitize_spaces_pem(opt.get_secret_value())
```

Comment:
I'd systematically put the parameter in the dict for consistency.

Also, does it make sense to have only the key or the cert ? Shouldn't we check that if one is specified, the other one should be specified as well ?